Formula/snapraid.rb

```diff
@@ -1,8 +1,8 @@
 class Snapraid < Formula
   desc "Backup program for disk arrays"
   homepage "https://snapraid.sourceforge.io/"
```

Comment:
May want to update homepage to avoid multiple redirects current homepage has
```suggestion
  homepage "https://www.snapraid.it/"
```